### PR TITLE
Bump vavr

### LINKED ISSUE #2246: Please consider updating the vavr version
The vavr version used by `janusgraph-cql` module is `0.9.0`. The newer versions of the vavr introduced a non-backward compatible change where they removed a method used by janusgraph as apart of this PR
https://github.com/vavr-io/vavr/pull/2197

This causes library conflicts when used  other libs which depends on newer versions of vavr.

The following method
```
CQLKeyColumnValueStore#getSlice(org.janusgraph.diskstorage.keycolumnvalue.KeySliceQuery, org.janusgraph.diskstorage.keycolumnvalue.StoreTransaction)
```
uses
```
io.vavr.concurrent.Future#fromJavaFuture(java.util.concurrent.ExecutorService, java.util.concurrent.Future<T>)
```
which has been removed from the newer versions of vavr

### PR DESCRIPTION
**Pull request by farodin91**
Fixes #2246

Signed-off-by: Jan Jansen <jan.jansen@gdata.de>

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there an issue associated with this PR? Is it referenced in the commit message?
- [x] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [x] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you written and/or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](https://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [ ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?
- [ ] If this PR is a documentation-only change, have you added a `[doc only]`
  tag to the first line of your commit message to avoid spending CPU cycles in
  Travis CI when no code, tests, or build configuration are modified?

### Note:
Please ensure that once the PR is submitted, you check Travis CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: janusgraph-core/src/main/java/org/janusgraph/graphdb/util/PriorityQueue.java [hunk 1]
```diff
@@ -79,7 +79,7 @@ public PriorityQueue(int maxSize, boolean prepopulate) {
 
   /** Determines the ordering of objects in this priority queue.  Subclasses
    *  must define this one method.
-   *  @return <code>true</code> iff parameter <tt>a</tt> is less than parameter <tt>b</tt>.
+   *  @return <code>true</code> iff parameter <code>a</code> is less than parameter <code>b</code>.
```
Comment (mbrukman) [inline janusgraph-core/src/main/java/org/janusgraph/graphdb/util/PriorityQueue.java:82]: The change from `<tt>` to `<code>` seems to be unrelated to the stated goal of bumping the version of vavr, but the change itself is fine for consistency, so no need to factor this out, unless you didn't mean to commit this delta.

**Review by mbrukman — APPROVED** [review]
LGTM

File: janusgraph-core/src/main/java/org/janusgraph/graphdb/util/PriorityQueue.java (hunk 1 above)
Comment (farodin91) [reply janusgraph-core/src/main/java/org/janusgraph/graphdb/util/PriorityQueue.java:82]: I did execute JanusGraph yesterday with java 11 during my try to test these changes. It failed since tt is deprecated.
